src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -619,15 +636,35 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
     }
 
     err = ConfigureAdvertisingData();
-    SuccessOrExit(err);
+    // SuccessOrExit(err);
 
     mFlags.Clear(Flags::kRestartAdvertising);
 
     sl_wfx_mac_address_t macaddr;
     wfx_get_wifi_mac_addr(SL_WFX_STA_INTERFACE, &macaddr);
 
+    rsi_ble_req_adv_t ble_adv = { 0 };
+
+    ble_adv.status = RSI_BLE_START_ADV;
+
+    ble_adv.adv_type         = RSI_BLE_ADV_TYPE;
+    ble_adv.filter_type      = RSI_BLE_ADV_FILTER_TYPE;
+    ble_adv.direct_addr_type = RSI_BLE_ADV_DIR_ADDR_TYPE;
+    rsi_ascii_dev_address_to_6bytes_rev(ble_adv.direct_addr, (int8_t *) RSI_BLE_ADV_DIR_ADDR);
```

Comment:
Can this function change the values of `adv_int_min` and `adv_int_max` ? If not the the two if X==0 are always true.

Reply:
removed the condition